### PR TITLE
versio upgradation changes

### PR DESCRIPTION
**Pull request by indraniBan**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.  

# Type of change  

- [ ] Bug fix  
- [ ] New feature  
- [x] Enhancement  
- [ ] Refactoring  
- [ ] Documentation  
- [ ] Other ( please specify )  

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/java/com/iemr/common/config/SwaggerConfig.java
Comment (drtechie) [inline src/main/java/com/iemr/common/config/SwaggerConfig.java:1]: This file is fully commented out. What happens to Swagger?

**Comment by drtechie** [comment]
@indraniBan LGTM, except for Swagger file commented out.

File: src/main/java/com/iemr/common/config/SwaggerConfig.java
Comment (indraniBan) [reply src/main/java/com/iemr/common/config/SwaggerConfig.java:1]: @drtechie now we are using one dependency for swagger so this config file is not required.

Comment (drtechie) [reply src/main/java/com/iemr/common/config/SwaggerConfig.java:1]: In that case, let's remove the file rather than commenting out the contents.

Comment (drtechie) [reply src/main/java/com/iemr/common/config/SwaggerConfig.java:1]: Also can you point me to the dependency for Swagger added?
Please confirm Swagger endpoint is working correctly too.

**Review by drtechie — APPROVED** [review]

**Pull request merged**